receiver/scrapererror/aliases.go

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+// Package scrapererror provides aliases for package scraper/scrapererror.
+// deprecated as of [v0.115.0]
```

Comment:
This needs to follow [the deprecated directive format](https://go.dev/wiki/Deprecated) so that linters pick it up
```suggestion
// Deprecated: [v0.115.0] Use `scraper/scrapererror` instead.
```